.travis.yml

```diff
@@ -23,7 +22,8 @@ before_script:
   # Deactivate xdebug if we don't do code coverage
   - if [ -z "$COVERAGE" ]; then rm -rfv /home/travis/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini ; fi
   - composer install
-
+  - composer update
```

Comment:
This should not be here, instead composer.lock file should be included in this PR.